.github/workflows/track_minetest.yml

```diff
@@ -33,14 +33,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       # Checkout minetest
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
         if: ${{ env.DID_PUSH == 'true' }}
         with:
           repository: '${{ github.repository_owner }}/minetest'
           ref: ${{ env.MINETEST_REV }}
           path: 'minetest'
       # Checkout irrlicht
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
         if: ${{ env.DID_PUSH == 'true' }}
         with:
           repository: '${{ github.repository_owner }}/irrlicht'
```

Comment:
what about `benchmark-action/github-action-benchmark`?

Reply:
i was not sure why it was pinned at a certain commit, additionally, there wasnt any warnings about it: https://github.com/minetest/benchmarks/actions/runs/7940706348 so I didnt dig into it further

Reply:
ah well ok